frameworks/Swift/vapor/Sources/TfbCommon/World.swift

```diff
@@ -0,0 +1,10 @@
+import Random
+
+public struct WorldMeta {
+  private init() { }
+
+  public static let maxId: UInt32 = 10000
+  public static let randomId = { () -> UInt32 in UInt32(1) + CryptoRandom.uint32 % maxId }
```

Comment:
Unless we need a cryptographically secure random number it might be better to use `PseudoRandom`. (will probably be a lot faster)